Writerside/topics/requirements.md

```diff
@@ -27,58 +27,61 @@ Requirement ID = `{TYPE}-{COMPONENT}-{PRIORITY}{NUMBER}`
-| **UC-VW-N1.1** | Explore and visualize the GTFS schedule.                                                                                                                                                                                                                              |
-| **UC-VW-N1.2** | Visualize the estimated vehicle positions from the GTFS realtime feed.                                                                                                                                                                                                |
-| **UC-VW-N1.3** | Display lines with departure times at selected stations.                                                                                                                                                                                                              |
-| **UC-VW-N1.4** | Query connections using the Raptor-Router and display the results on a map.                                                                                                                                                                                           |
-| **UC-VW-N1.5** | Query reachability from a station within a given time budget.                                                                                                                                                                                                         |
-| **UC-VW-N1.6** | Provide an option to visualize the functioning of the Raptor algorithm by visualizing the Raptor rounds until the target stop is reached.                                                                                                                             |
+| ID             | Done | Description                                                                                                                                                                                                                                                           |
+|----------------|------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **UC-SE-N1**   | ❌    | Implement user accounts with login, which allows saving favorite connections in a database.                                                                                                                                                                           |
+| **NF-SE-N1**   | ❌    | Horizontal scaling: Store the parsed static GTFS schedule and built Raptor data structures with a validity timestamp to a common storage (mounted volume or database), which is used by new instances of the service when started, instead of parsing the GTFS again. |
```

Comment:
@munterfi I would reomove the "title" `Horizontal scaling:` since one line below, there is again a topic about `Horizontal scaling`